<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 47, 48);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);
</string>
     </property>
     <property name="text">
      <string>Input data for a new flight:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);</string>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="numberLine">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
gridline-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);
border-color: rgb(182, 182, 182);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);</string>
     </property>
     <property name="text">
      <string>Aim:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="aimLine">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
gridline-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);
border-color: rgb(182, 182, 182);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);</string>
     </property>
     <property name="text">
      <string>Cost:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="costLine">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
gridline-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);
border-color: rgb(182, 182, 182);</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 47, 48);
font: 10pt &quot;Constantia&quot;;
color: rgb(182, 182, 182);</string>
     </property>
     <property name="text">
      <string>Time and date:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(182, 182, 162);
background-color: rgb(46, 47, 48);</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttons">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50, 50, 50);
selection-background-color: rgb(74, 75, 77);
font: 10pt &quot;Constantia&quot;;
font-color: rgb(193, 195, 196);
color: rgb(182, 182, 182);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>AddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>AddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
